.travis.yml

```diff
@@ -105,3 +110,34 @@ jobs:
               while read LINE; do travis_retry gpg --keyserver hkp://subset.pool.sks-keyservers.net --recv-keys $LINE; done < contrib/verify-commits/trusted-keys &&
               travis_wait 50 contrib/verify-commits/verify-commits.py;
           fi
+    - stage: lint BSD
+      os: osx
+      # Use macOS consistent with the minimum supported version: 10.11
```

Comment:
If you mean our next supported minimum macOS version, shouldn't this be 10.10?

Reply:
Thanks - was relying on faulty memory. Also it happens to be the earliest version supported by Travis. Updated. #13617